registry-requirements.md

```diff
@@ -120,7 +120,19 @@ What does the binding have to contain to go into the table
     - Implementing a non-commercial Consumer application/driver
     - Conditions for commercial use: E.g. building a commercial product with the binding
     - Making a statement about your product's supporting that binding
-9. DISCUSS: What is the objective mechanism to confirm the status change from "initial" to "current"? Should we have a test suite? Plugfest experience?
+9. Transition from "Initial" to "Current"
+  - Starting from the initial submission each binding has to demonstrate a certain level of concrete development maturity. This process involves real-world testing, which can take place in Plugfests, independent testing events, or even informal collaboration between developers. These testing events do not have to be organized by W3C and can be conducted remotely, including over VPN. The goal is to demonstrate that the binding correctly maps protocol operations and is well understood by at least two parties.
+  - At each testing event every operation defined in the binding MUST be validated automatically (e.g., scripts, test suites, etc.) and the results SHOULD be published in a dedicated document (README, or other human-readable documents) called *Test Report*.
+  - A *Test Report*
+      - MUST contain information on the testing environment
+      - MUST provide an example of the logical process (not necessarily code) about how a TD can be processed to establish a communication between consumer and exposer.
+      - MUST explain where discussions on implementation experience should be collected
+      - SHOULD provide the history of all the past events (or explain how to retrieve the history of the results gathered during those events)
+      - SHOULD contain a reference to possible implementations of Consumers or Exposers. 
+  - For the binding to transition to the "Current" state, the test report MUST contain at least one implementation of a Consumer (capable of understanding and performing all the operations described in the binding) and one Exposer (capable of creating TDs with all the operations and features described in the binding). Additional implementations can be added even after the transition to the Current
```

Comment:
I would not say that Exposer should be able to create TDs. TDs can come from another entity than the Thing